README.md

```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
 
 
 ### Using Virtual Environment
-1. First set up a virtualenv
+1. First set up a virtualenv. Fabrik runs on Python2.7.
     ```
     sudo apt-get install python-pip python-dev python-virtualenv
-    virtualenv --system-site-packages ~/Fabrik
+    virtualenv --system-site-packages ~/Fabrik --python=python2.7
     source ~/Fabrik/bin/activate
     ```
 
-2. Clone the repository
+2. Clone the repository via git
     ```
-    git clone --recursive https://github.com/Cloud-CV/Fabrik.git
+    git clone --recursive https://github.com/Cloud-CV/Fabrik.git && cd Fabrik
     ```
 
 3. Rename settings/dev.sample.py as settings/dev.py and change credential in settings/dev.py
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
-    Replace the hostname to ``` localhost ``` in settings/dev.py line 14.
+    * Replace the hostname to ``` localhost ``` in settings/dev.py line 15.
+    It should now look like this:  
+    ```
+    'HOST': os.environ.get("POSTGRES_HOST", 'localhost'), 
+    ```
 
-4. Install redis server and replace the hostname to ``` localhost ``` in settings/common.py line 99.
+4. Install redis server  
     ```
     sudo apt-get install redis-server
     ```
-    Replace celery result backend in settings/common.py line 122 with
+    * Replace the hostname to ``` localhost ``` in settings/common.py line 115.
+
+    ```
+    "CONFIG": {
+        # replace redis hostname to localhost if running on local system
+        "hosts": [("localhost", 6379)],
+        "prefix": u'fabrik:',
+        },
     ```
-    CELERY_RESULT_BACKEND = 'redis://redis:6379/0'
+
+    * Replace celery result backend in settings/common.py line 122 with localhost.
+    ```
+    CELERY_RESULT_BACKEND = 'redis://localhost:6379/0'
     ```
-    Replace celery broker url and result backend hostname to ``` localhost ``` in ide/celery_app.py line 8 with
+    * Replace celery broker url and result backend hostname to ``` localhost ``` in ide/celery_app.py, line 8.
     ```
-    broker='redis://redis:6379/0'
-    backend='redis://redis:6379/0'
+    app = Celery('app', broker='redis://localhost:6379/0', backend='redis://localhost:6379/0', include=['ide.tasks'])
     ```
```

Comment:
Also add a new line. It doesn't look good in markdown

Reply:
Should I add new lines before every code spans throughout the markdown? It will look more readable that way.